pages/scripts/delayed.js

```diff
@@ -81,41 +78,17 @@ async function OptanonWrapper() {
     /* eslint-enable */
   };
   if (!localStorage.getItem('OptIn_PreviousPermissions')) {
-    const adobeSettings = OneTrustActiveGroup();
-    adobeSettings.tempImplied = true;
-    localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(adobeSettings));
+    const settings = OneTrustActiveGroup();
+    settings.tempImplied = true;
+    localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(settings));
   }
 
-  clearDataLayer();
-  loadScript(`https://assets.adobedtm.com/d17bac9530d5/a14f7717d75d/launch-aa66aad171be${isProd ? '.min' : ''}.js`, () => {
```

Comment:
Looks like they called Gigya after launch was added previously.

Is there any reason we should do this also for GTM?

Reply:
This was done to pass user info in the now removed sendPageEvents()

Reply:
Removed, we don't need the gigya id in the datalayer.